src/core/lwt_result.ml

```diff
@@ -22,12 +23,13 @@ let map f e =
       | Ok x -> Ok (f x))
     e
 
-let map_err f e =
+let map_error f e =
   Lwt.map
     (function
       | Error e -> Error (f e)
       | Ok x -> Ok x)
     e
+let map_err f e = map_error f e
```

Comment:
Why don't we mark this as deprecated as well?

Reply:
Isn't the deprecation warning in the interface sufficient?